shaazk/Free-Mentors
# #168111205 Fix signup endpoint bugs

#### What does this PR do?
- it fixes signup endpoint 
#### Description of Task to be completed?
- more than one user can be added
#### How should this be manually tested?
 -clone free mentors on your computer and open the older in your VS Code or any other html IDE. in your terminal, run nodemon by typing npm start and then open your postman and run http://localhost:3000/api/v1/signup under POST function and in body, insert credentials and click on send
- in your VSCode after running nodemon, you can click ctrl + the link in the terminal where you see server running on http://localhost:3000
#### What are the relevant pivotal tracker stories?
- [#168039463](https://www.pivotaltracker.com/story/show/168039463) 
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/21080338/63705283-156c1880-c82d-11e9-9305-f97c345f972e.png)
![image](https://user-images.githubusercontent.com/21080338/63705416-5a904a80-c82d-11e9-887c-0d9895c3ca0a.png)
